lib/listener-tracker.js

```diff
@@ -9,12 +9,6 @@ function EventEmitterWrapper(target) {
 }
 
 EventEmitterWrapper.prototype = {
-    _onProxy: function(type, event, listener) {
```

Comment:
this makes sense if it were being used more than once, but it was only being used for `on`, which made it weird to put `addEventListener` here where it technically could be other types other than `on` in the future. So I moved it down to the `on` method itself. If there are more types added in the future, it could be added back, but seems like it was more "speculative generality".